pages/form/Question2.jsx

```diff
@@ -2,9 +2,9 @@ import { useState } from "react";
 import {
   Box,
   Grid,
-  Heading,
+  Heading, 
   Text,
-  Select,
+  Select, 
```

Comment:
please remove trailing space

```suggestion
  Select, 
```